pages/linux/pmap.md

```diff
@@ -0,0 +1,20 @@
+# pmap
+
+> Report memory map of a process or processes.
```

Comment:
But the command applies to one process at time.
```suggestion
> Report memory map of a process.
```

Reply:
The command can be used on more processes as such:

`pmap {{pid1}} {{pid2}}`

However I will commit this suggestion as it is not a common use case.

Reply:
add an example for this.

Reply:
Done